### PR TITLE
Build the Contact section

### PR DESCRIPTION
**Pull request by hienphan0111**
- Make the contact form in the footer.

### CODE REVIEW

**Review by Tufahel — CHANGES_REQUESTED** [review]
Hi @hienphan0111 @Uchenna-ibute :wave:  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 

## Invalid Code Review Request

You have submitted a project, but you did not add linter files (`.stylelintrc.json`, `.hintrc`) for the linters check rather, you ignored the `.stylelintrc.json` file in `.gitignore`. Kindly add those files and consider not putting them inside `.gitignore` as we can't be sure you have edited those `.linter` files or not if those are on `.gitignore`.



- Kindly add linters following [Microverse linters guidelines ](https://github.com/microverseinc/linters-config/tree/master/html-css)and apply for another review.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: styles.css [hunk 1]
```diff
@@ -382,4 +439,11 @@ h1 {
...
     overflow: hidden;
   }
+
+  .contact {
+    background-image: url('./image/contact-form-shapes-mobile.png');
+    background-repeat: no-repeat;
+    background-position: right top;
+    z-index: 2;
+  }
 }
```
Comment (levy002) [inline styles.css]: - [ ] Kindly make sure your background is visible in order to look like the provided Figma design. :wink: 

   - ### YOUR DESIGN
    image


   - ### FIGMA DESIGN 
   
     image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @hienphan0111   , :wave:
### I'm Levy, your PR reviewer

Excellent work :sunglasses: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi @hienphan0111 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
68747470733a2f2f692e67696665722e636f6d2f6f726967696e2f36352f36353164353034326534396532656561363365613934633039653131393731365f773230302e676966


## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. 🙏
 - [ ]  Correct GitHub Flow 👍🏾
 - [ ]  Good commit history 🥇
 - [ ]  Good PR title/description 💯
 - [ ]  All linters passed 🥇
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**